selfdrive/modeld/navmodeld.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+import os
+import gc
+import math
+import time
+import ctypes
+import numpy as np
+from pathlib import Path
+from typing import Tuple, Dict
+from cereal import messaging
+from cereal.messaging import PubMaster, SubMaster
+from cereal.visionipc import VisionIpcClient, VisionStreamType
+from openpilot.system.hardware import TICI
+from openpilot.system.swaglog import cloudlog
+from openpilot.common.params import Params
+from openpilot.common.realtime import set_realtime_priority
+from openpilot.selfdrive.modeld.constants import IDX_N
+from openpilot.selfdrive.modeld.models.commonmodel_pyx import Runtime
+
+USE_SNPE = int(os.getenv('USE_SNPE', str(int(TICI))))
+if USE_SNPE:
+  os.environ['ADSP_LIBRARY_PATH'] = "/data/pythonpath/third_party/snpe/dsp/"
```

Comment:
Again, this used to be specified in the navmodeld launch script but now needs to be specified elsewhere. This seemed like the most appropriate place to me.

Reply:
lgtm, though can we put the ADSP_LIBRARY_PATH setting in snpemodel_pyx and move this if statement into a helper since it'll be in all the model runners?